.eslintrc

```diff
@@ -0,0 +1,80 @@
+{
+  "env": {
```

Comment:
Are you sure we want to enable all of these different environments at the top level? Or should we enable them in the specific sub directories that they are relevant in?

Reply:
That's right, we should make set them at sub-directory level, Nice one!